opentech/static_src/src/app/src/SubmissionsByRoundApp.js

```diff
@@ -28,27 +29,22 @@ class SubmissionsByRoundApp extends React.Component {
         return (
             <>
                 <Switcher selector='submissions-by-round-app-react-switcher' open={this.state.detailOpen} handleOpen={this.detailOpen} handleClose={this.detailClose} />
+
                 <div style={this.state.style} ref={this.setOriginalContentRef} dangerouslySetInnerHTML={{ __html: this.props.pageContent }} />
+
                 {this.state.detailOpen &&
-                    <SubmissionsGrid>
-                        <div>
-                            {this.renderSubmissionsByRound()}
-                        </div>
-                        <div>
-                            <h4>Column 2</h4>
-                        </div>
-                        <div>
-                            <h4>Column 3</h4>
-                        </div>
-                    </SubmissionsGrid>
+                    <DetailView>
+                        {this.renderSubmissionsByRound()}
+                        <DisplayPanel />
+                    </DetailView>
                 }
             </>
         )
     }
 
-  renderSubmissionsByRound() {
-    return <SubmissionsByRoundContainer roundId={this.props.roundId} />;
-  }
+    renderSubmissionsByRound() {
```

Comment:
Does this need to be a separate function? `<SubmissionsByRoundContainer>` could just be rendered directly within <SubmissionsGrid>, much like your change in e54945b12ad3397256f24914245e4ff95737db01